Packages/PackageSources/meridian-simcache/InGamePanels/InGamePanel_SimCache.xml

```diff
@@ -1,23 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<SimBase.Document Type="InGamePanels" version="1.0">
+<SimBase.Document Type="InGamePanels" version="1,0">
```

Comment:
As I recall, when inspecting the default InGamePanels, they use `1,0` instead of `1.0`, so I made that change for consistency.  It seemed to _run_ with `1.0` before, but `1,0` sounds more correct (at least as far as being in line with the defaults)?